go.sum

```diff
@@ -2232,8 +2232,8 @@ github.com/otiai10/copy v1.14.0/go.mod h1:ECfuL02W+/FkTWZWgQqXPWZgW9oeKCSQ5qVfSc
 github.com/pascaldekloe/goe v0.0.0-20180627143212-57f6aae5913c/go.mod h1:lzWF7FIEvWOWxwDKqyGYQf6ZUaNfKdP144TG7ZOy1lc=
 github.com/pelletier/go-toml v1.2.0/go.mod h1:5z9KED0ma1S8pY6P1sdut58dfprrGBbd/94hg7ilaic=
 github.com/pelletier/go-toml v1.7.0/go.mod h1:vwGMzjaWMwyfHwgIBhI2YUM4fB6nL6lVAvS1LBMMhTE=
-github.com/percona/everest-operator v0.6.0-dev1.0.20250131090446-40b6d1d65b10 h1:LI/o3pAYQdC+5h50MPbR7q4L7kMcuPN0TlsoiYxi4lc=
-github.com/percona/everest-operator v0.6.0-dev1.0.20250131090446-40b6d1d65b10/go.mod h1:jpmlzDw0avyNWwmlBABbaHNZO4/G3q9AonI1GoXfQfE=
+github.com/percona/everest-operator v0.6.0-dev1.0.20250212204444-6db98d6f0f21 h1:NvatZdUUBmUOFOxgiIG4GhijBLgSo6Z7Y18y1+8zCSg=
+github.com/percona/everest-operator v0.6.0-dev1.0.20250212204444-6db98d6f0f21/go.mod h1:jpmlzDw0avyNWwmlBABbaHNZO4/G3q9AonI1GoXfQfE=
```

Comment:
Leaving a comment to not forget to update `everest-operator` dependency when the operator PR is merged